### PR TITLE
Feature/views

### PR DESCRIPTION
**Pull request by lili2311**
- Created most of the views, urls 
- did some fixes to forms and deleted some repeated code and files

### CODE REVIEW

File: pimpmycause/core/views.py [hunk 1]
```diff
@@ -24,8 +24,42 @@ class AboutView(TemplateView):
...
+    """
+    The About Us View.
+    """
+    template_name = "core/who-we-are.html"
+
+
+
+class HowItWorks(TemplateView):
+    """
+    The About Us View.
```
Comment (Icicleta) [inline pimpmycause/core/views.py]: is this the right description?

File: pimpmycause/core/views.py [hunk 2]
```diff
@@ -24,8 +24,42 @@ class AboutView(TemplateView):
     template_name = "core/about.html"
 
 
+class WhoWeAre(TemplateView):
+    """
+    The About Us View.
```
Comment (Icicleta) [inline pimpmycause/core/views.py]: right description?

File: pimpmycause/core/views.py [hunk 3]
```diff
@@ -24,8 +24,42 @@ class AboutView(TemplateView):
...
+
+class OurPartners(TemplateView):
+    """
+    Meet The Team View.
+    """
+    template_name = "core/our-partners.html"
+
+class BecomeAPartner(TemplateView):
+    """
+    Meet The Team View.
```
Comment (Icicleta) [inline pimpmycause/core/views.py]: same.. description should be different on each right?

**Comment by lili2311** [comment]
Pushed a fix for view descriptions

**Comment by Icicleta** [comment]
👍

**Pull request merged**